README.md

```diff
@@ -75,6 +76,36 @@ GET /manifests
 GET /manifests/<name>
 ````
 
+## Determine whether partial file uploads are supported
+````
+OPTIONS /files
+````
+This returns the following header and body:
+````
+Server: ....
+Allow: GET,HEAD,POST,OPTIONS,TRACE
+Content-Type: Application/octet-binary
```

Comment:
This should probably be the Accept header, as application/octet-stream. Content-Type would be application/json here I think.